expression/bench_test.go

```diff
@@ -523,6 +523,33 @@ func (g *randDurInt) gen() interface{} {
 	return int64(rand.Intn(types.TimeMaxHour)*10000 + rand.Intn(60)*100 + rand.Intn(60))
 }
 
+// locationGener is used to generate location
+type locationGener struct{}
+
+func (g *locationGener) gen() interface{} {
+	var locations = []string{
+		usaLocation,
+		jisLocation,
+		isoLocation,
+		eurLocation,
+		internalLocation,
+	}
+	return locations[rand.Intn(5)]
+}
+
+// formatGener is used to generate format
```

Comment:
is used to generate a format for the built-in function `GetFormat`.

Reply:
ok

Reply:
we can use `timeFormatGener`

Reply:
yep